### PR TITLE
Ko-2770

### PR DESCRIPTION
**Pull request by Ken-tech-max**
fixed: 2770

### CODE REVIEW

File: packages/ui/src/memberships/components/MemberProfile/MemberProfile.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ import { MemberModal } from './MemberModal'
 import { MemberSideRoles } from './MemberRoles'
 import { MemberModalCall } from './types'
 
-type ProfileTabs = 'DETAILS' | 'ACCOUNTS' | 'ROLES'
+type ProfileTabs = 'DETAILS' | 'ACCOUNTS' | 'Working Group Roles'
```
[the remark this answers is not in the document]
Comment (Ken-tech-max) [reply packages/ui/src/memberships/components/MemberProfile/MemberProfile.tsx:22]: Alright I will change

File: packages/ui/src/proposals/components/ProposalDetails/renderers/Markdown/components/OpeningDescriptionPreview.tsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ export const OpeningDescriptionPreview = ({ onClose, description }: DescriptionP
       <SidePane topSize="s">
         <SidePaneHeader>
           <SidePanelTop>
-            <SidePaneTitle>Opening Description</SidePaneTitle>
+            <SidePaneTitle>Signal Description</SidePaneTitle>
```
Comment (singulart) [inline packages/ui/src/proposals/components/ProposalDetails/renderers/Markdown/components/OpeningDescriptionPreview.tsx:35]: Please explain this change and it's purpose?

File: packages/ui/src/proposals/modals/AddNewProposal/components/SpecificParameters/WorkingGroupLeadOpening/CreateWorkingGroupLeadOpening/StakingPolicyAndReward.tsx [hunk 3]
```diff
@@ -13,6 +13,8 @@ export const StakingPolicyAndReward = () => {
   const { watch } = useFormContext()
   const leavingUnstakingPeriod = watch('stakingPolicyAndReward.leavingUnstakingPeriod')
 
+  const maxAllowedForStakingAmount = 200
```
Comment (singulart) [inline packages/ui/src/proposals/modals/AddNewProposal/components/SpecificParameters/WorkingGroupLeadOpening/CreateWorkingGroupLeadOpening/StakingPolicyAndReward.tsx:16]: Please explain this change and it's purpose?

File: packages/ui/src/proposals/modals/AddNewProposal/components/SpecificParameters/WorkingGroupLeadOpening/CreateWorkingGroupLeadOpening/StakingPolicyAndReward.tsx [hunk 4]
```diff
@@ -30,13 +32,23 @@ export const StakingPolicyAndReward = () => {
             units="tJOY"
             tight
             name="stakingPolicyAndReward.stakingAmount"
+            validation={stakingAmount < maxAllowedForStakingAmount ? 'invalid' : undefined}
```
Comment (singulart) [inline packages/ui/src/proposals/modals/AddNewProposal/components/SpecificParameters/WorkingGroupLeadOpening/CreateWorkingGroupLeadOpening/StakingPolicyAndReward.tsx:35]: Please explain this change and it's purpose?

File: packages/ui/src/proposals/modals/AddNewProposal/components/SpecificParameters/WorkingGroupLeadOpening/CreateWorkingGroupLeadOpening/StakingPolicyAndReward.tsx [hunk 5]
```diff
@@ -30,13 +32,23 @@ export const StakingPolicyAndReward = () => {
...
+                : undefined
+            }
           >
             <InputNumber
               id="staking-amount"
               isTokenValue
               placeholder="0"
               name="stakingPolicyAndReward.stakingAmount"
               isInBN
+              maxAllowedValue={maxAllowedForStakingAmount}
```
Comment (singulart) [inline packages/ui/src/proposals/modals/AddNewProposal/components/SpecificParameters/WorkingGroupLeadOpening/CreateWorkingGroupLeadOpening/StakingPolicyAndReward.tsx:48]: Please explain this change and it's purpose?

**Review by singulart — CHANGES_REQUESTED** [review]

**Pull request closed without merging**